doc/data/messages/o/overridden-final-method/bad.py

```diff
@@ -0,0 +1,12 @@
+from typing import final
+
+
+class Base:
+    @final
+    def my_method(self):
+        pass
+
+
+class Subclass(Base):
+    def my_method(self):  # [overridden-final-method]
+        pass
```

Comment:
```suggestion
class Animal:
    @final
    def breath(self):
        return True


class Cat(Animal):
    def breath(self):  # [overridden-final-method]
        pass
```

Reply:
Quite an intuitive example @Pierre-Sassoulas.
I've changed the naming slightly.